### PR TITLE
chore: kubevirt-compiled support on windows

### PR DESCRIPTION
**Pull request by zirain**
`:` is invalid on windows, make this repo cloneable on windows.

### CODE REVIEW

File: examples/kubevirt/README.md [hunk 1]
```diff
@@ -35,4 +35,5 @@ Here are the steps for updating kubevirt-operator:
   rm -rf kubevirt-compiled
   nomos hydrate --path=kubevirt --source-format=unstructured --output kubevirt-compiled
   addlicense -v -c "Google LLC" -f ../LICENSE_TEMPLATE -ignore=vendor/** kubevirt-compiled 2>&1 | sed '/ skipping: / d'
+  mv kubevirt-compiled/clusterrole_kubevirt.io:operator.yaml kubevirt-compiled/clusterrole_kubevirt.io-operator.yaml
```
Comment (sdowell) [inline examples/kubevirt/README.md:38]: Could you add a note/comment in the doc here explaining why the file needs to be renamed?

**Review by sdowell — COMMENTED** [review]
/ok-to-test

**Comment by sdowell** [comment]
The failures in test groups 2/3 look to be known metrics flakes, but the failure in test group 1 looks like a genuine failure:
```
  cli_test.go:573: Only in ../../examples/kubevirt-compiled: clusterrole_kubevirt.io-operator.yaml
        Only in /tmp/nomos-e2e/test-nomos-hydrate-with-unknown-scoped-object2598646333/compiled-without-api-server-check: clusterrole_kubevirt.io:operator.yaml
        
    cli_test.go:574: ERROR: exit status 1
    cli_test.go:591: Only in ../../examples/kubevirt-compiled: clusterrole_kubevirt.io-operator.yaml
        Only in /tmp/nomos-e2e/test-nomos-hydrate-with-unknown-scoped-object2598646333/compiled-with-api-server-check: clusterrole_kubevirt.io:operator.yaml
        
    cli_test.go:592: ERROR: exit status 1
=== CONT  TestRevertClusterRole
```

**Comment by sdowell** [comment]
FYI I put up a similar PR: https://github.com/GoogleContainerTools/kpt-config-sync/pull/554

**Comment by zirain** [comment]
replaced by https://github.com/GoogleContainerTools/kpt-config-sync/pull/554

**Comment by zirain** [comment]
thanks @sdowell for fix this.

**Pull request closed without merging**